.ci/docker/test/verify.sh

```diff
@@ -88,21 +102,19 @@ if [[ -z "${actualLabels}" || "${actualLabels}" == "null" || "${actualLabels}" =
 fi
 
 # Generate the expected labels files with the dynamic properties substituted
-labelsGoldenFile="${BASH_SOURCE%/*}/docker-labels.golden.json"
+labelsGoldenFile="${BASH_SOURCE%/*}/$GOLDENFILE"
 expectedLabels=$(
   jq --sort-keys -n \
     --arg VERSION "${VERSION}" \
     --arg REVISION "${REVISION}" \
     --arg DATE "${DATE}" \
-    --arg BASE_IMAGE "${BASE_IMAGE}" \
-    --arg BASEDIGEST "${DIGEST}" \
```

Comment:
No because this was/is the difference of the scripts the variables have just named differently. But it is still checked. https://github.com/camunda/zeebe/blob/1538ae70a1dbe55c04a1758ad067b5a11de7dad4/.ci/docker/test/verify.sh#L111

That reminds me I have to rename it on the operate golden file


Operate
https://github.com/camunda/zeebe/blob/1538ae70a1dbe55c04a1758ad067b5a11de7dad4/.ci/docker/test/operate-docker-labels.golden.json#L10

Zeebe

https://github.com/camunda/zeebe/blob/1538ae70a1dbe55c04a1758ad067b5a11de7dad4/.ci/docker/test/zeebe-docker-labels.golden.json#L8

Reply:
Fixed https://github.com/camunda/zeebe/pull/17816/commits/459253c8934cc406ea5baf59784eb74240521a43